src/main/java/items/Items.java

```diff
@@ -0,0 +1,112 @@
+package items;
+import java.io.IOException;
+import java.util.ArrayList;
+
+import handler.DataHandler;
+import msg.StdMsgs;
+import task.Task;
+import msg.Msg;
+
+
+/**
+ * The items class is a representation of a list of tasks that is able to add, delete, mark and unmark tasks on that
+ * list and return an appropriate response
+ */
+public class Items {
+    /** tracks number of tasks created*/
+    private int item_count = 0;
+    /** ArrayList<task> to store tasks */
+    private ArrayList<Task> item_list = new ArrayList<Task>();
```

Comment:
Would it enhance code readability if you use plural form to represent the list of items (eg: `items` )?